src/electron/get_electron_build_parameters.mjs

```diff
@@ -25,7 +25,7 @@ const ELECTRON_PLATFORMS = ['linux', 'windows'];
   => an object containing the required electron parameters.
 */
 export function getElectronBuildParameters(parameters) {
-  const {platform, buildMode, stagingPercentage, sentryDsn} = getBuildParameters(parameters);
```

Comment:
The staging percent logic is better here than in`getBuildParameters`, since it's electron-specific.

But I think it should really be where the publication code is.